### DISCUSSION TITLE
Fingerprint reader without dialog

### DISCUSSION THREAD

**Discussion by ruisilva450** [NiklasMerz/cordova-plugin-fingerprint-aio#147] [Ideas]
#### Feature Description
Hi,
I think it would be awesome if the plugin didn't needed to show the modal at all and instead leave the UI to the developer.
As an example we have the Revolut app.
The ideal flow would be the same as .show(options) but without "showing" anything:
- Call FingerprintAIO.read().then(callback,error)

**Comment by NiklasMerz** [comment]
Sounds interesting. Do you think about Android or iOS. At least for iOS I don´t think it is possible.

If anybody needs this feel free to implement it.

**Comment by ruisilva450** [comment]
I have come from Xamarin where on Android you can provide this type of interaction (or lack of 😃).
[more info](https://github.com/smstuebe/xamarin-fingerprint/issues/81)
